products/opensuse/main.pm

```diff
@@ -437,7 +437,8 @@ sub load_consoletests {
         loadtest "console/mysql_srv";
         loadtest "console/dns_srv";
         if (!is_staging) {
-            loadtest "console/mysql_odbc";
+            # disabled until further investigation
+            # loadtest "console/mysql_odbc";
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit.

I recommend you just remove the test module again and instead create a progress ticket which you assign to your self to add the module back to openSUSE tests as soon as you found the time